data.sql

```diff
@@ -1,198 +1,11 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (
-  name, 
-  date_of_birth, 
-  escape_attempts, 
-  neutered, 
-  weight_kg ) VALUES (
-  'Agumon',
-  '2020-02-03',
-  0,
-  true,
-  10.23
-);
+INSERT INTO animals (name) VALUES ('Agumon'), ('Gabumon'), ('Pikachu'), ('Devimon'), ('Charmander'), ('Plantmon'), ('Squirtle'), ('Angemon'), ('Boarmon'), ('Blossom');
```

Comment:
- [ ] Kindly make sure you correctly address this project requirement 👇🏻 

![image](https://github.com/elad237/Vet-clinic-database/assets/79058364/4bd39bb8-0d01-4a39-9385-d492d71de35d)